src/zap/zapimport.cpp

```diff
@@ -362,12 +362,12 @@ void ZapImport::Save(ZapWriter * pZapWriter)
 {
     if (IsReadyToRunCompilation())
     {
-        SIZE_T value = 0;
+        target_size_t value = 0;
```

Comment:
It looks odd to use `target_size_t` for all these since none of them are actually sizes.

Would `TARGET_POINTER` or `TARGET_POINTER_TYPE` be a better name?

Reply:
The only reason I had in mind when picked `target_size_t` is to be consistent with clrjit where we already have `target_size_t` typedef. I renamed this to `TARGET_POINTER_TYPE` in src/zap